src/components/AddNewBookForm/AddNewBookForm.js

```diff
@@ -48,18 +45,18 @@ const AddNewBookFormBase = props => {
       .books()
       .add({
         title,
-        owner: JSON.parse(localStorage.getItem('authUser')).uid,
+        owner: props.authUser.username,
+        avatar: props.authUser.photoURL,
         rating: parseInt(rating),
         description,
         author,
         location,
         imageUrls,
-        type: 'lend',
+        type,
```

Comment:
Here must be a type, either lend or give

Reply:
I think this works fine like it is, since I have made some changes. I can show you tomorrow!